combustion-android-ble/src/main/java/inc/combustion/framework/service/DeviceManager.kt

```diff
@@ -76,7 +77,7 @@ class DeviceManager(
         val autoLogTransfer: Boolean = false,
         val meatNetEnabled: Boolean = false,
         val probeAllowlist: Set<String>? = null,
-        val messageTypeCallback: (UByte)-> NodeMessage? = { messageType: UByte -> NodeMessage.fromUByte(messageType) }
+        val messageTypeCallback: (UByte) -> NodeMessage? = { messageType: UByte -> NodeMessageType.fromUByte(messageType) }
```

Comment:
Switched from `NodeMessage` to `NodeMessageType` for `(UByte) -> NodeMessage?` - intended?

Reply:
ok, I see it is a supertype

Reply:
Yeah, and I think `NodeMessage.fromUByte()` wasn't actually the intended behavior.